__tests__/app.test.js

```diff
@@ -1,6 +1,33 @@
 const request = require("supertest");
 
 const app = require("../app.js");
+const db = require("../db/connection.js");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET: 200 -- responds with an array of all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
```

Comment:
consider how destructuring may be useful when accessing properties from the response object